PowerEditor/src/langs.model.xml

```diff
@@ -243,6 +243,8 @@
             <Keywords name="instre2"></Keywords>
             <Keywords name="type1"></Keywords>
         </Language>
+        <!-- User Defined Language -->
+        <Language name="udf"/>

```

Comment:
Hi, I guess it would in some way. But I've tried to figure out easiest way to be able to define specific tab settings for **UserDefinedLanguage**. And I've found out that "**udf**" is used internally already. It's part of  **Scitilla** with mean of "**User Define File**". So basically I used existed feature, and configured it to specific needs.
To be able to use "**udl**" (or anything different), it would take to do more coding, and also what is more important to change core **Scitilla** code. So this change would affect its future releases => need to update it all the time.
So I've decided to do it this way without any bigger impact on core code itself.